tests/x11/updates_xfce.pm

```diff
@@ -0,0 +1,19 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use testapi;
+
+sub run() {
+    x11_start_program("apper");
+    
```

Comment:
It was just first suggestion from rbrown
I will use updates from notification icon